src/Tests/Microsoft.NET.Publish.Tests/GivenThatWeWantToPublishASingleFileApp.cs

```diff
@@ -500,6 +499,45 @@ public void It_rewrites_the_apphost_for_non_single_file_publish()
             appHostSize.Should().BeLessThan(singleFileSize);
         }
 
+        [RequiresMSBuildVersionTheory("16.8.0")]
+        [InlineData("net6.0")]
+        public void ILLink_analyzer_warnings_are_produced(string targetFramework)
```

Comment:
Note: if this succeeds it's probably because we still have a reference to the old package, as opposed to the new analyzers working. I think https://github.com/mono/linker/issues/1745 is needed to fix the new analyzers.